source/features/collapse-duplicated-reviews.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	const singleReviewThreadComments = select.all('.review-comment.unminimized-comment:only-child');
+	for (const comment of singleReviewThreadComments) {
+		if (!select.exists(`.review-comment#${comment.id}:not(:only-child)`)) { // The review comment is not a duplicate
+			continue;
+		}
+
+		// Change the <details> element to a link to the original comment
+		const details = comment.closest('details')!;
+		const commentUrl = new URL(location.href);
+		commentUrl.hash = comment.id;
+		details.replaceWith(
+			<a
+				className="d-block py-2 px-3 mb-3 border rounded-2 color-bg-subtle text-mono text-small"
+				href={commentUrl.href}
+			>
+				#{comment.id}
```

Comment:
I'd rather show the file name, this doesn't make sense to a user

Reply:
How about an informative text instead, like "Duplicated comment. Click to jump to original thread"?

I just think showing a file name when this isn't a link to a file is a bit weird :thinking:

Reply:
Let we do in `short-repo-url`:

> file.tsx (comment)

Let's not overthink this 😁